src/components/clusterConfiguration/BasicNetworkFields.tsx

```diff
@@ -84,6 +96,56 @@ const getVipValidationsById = (
   }, {});
 };
 
+const SubnetHelperText: React.FC<{ matchingSubnet: HostSubnet; cluster: Cluster }> = ({
+  matchingSubnet,
+  cluster,
+}) => {
+  const excludedHosts =
+    cluster?.hosts?.filter(
+      (host) => !matchingSubnet.hostIDs.includes(host.requestedHostname || ''),
+    ) || [];
+
+  if (excludedHosts.length === 0) {
+    return null;
+  }
+
+  return (
+    <Alert
+      title="This subnet range is not available on all hosts"
+      variant={AlertVariant.warning}
+      isInline
+    >
+      Hosts outside of this range will not be included in the new cluster.
+      <br />
+      <Popover
+        position="right"
+        bodyContent={
+          <ul>
+            {excludedHosts
+              .sort(
+                (hostA, hostB) =>
+                  hostA.requestedHostname?.localeCompare(hostB.requestedHostname || '') || 0,
+              )
+              .map((host) => (
+                <li key={host.id}>{host.requestedHostname}</li>
+              ))}
+          </ul>
+        }
+        minWidth="30rem"
+        maxWidth="50rem"
+      >
+        <Button
```

Comment:
It would be better to use `actionLinks` prop + `AlertActionLink` instead of Button on the `Alert` but I am not sure if it works well with `Popover`.

Reply:
works fine, done